strutil/strutil.go

```diff
@@ -111,3 +111,41 @@ func TruncateOutput(data []byte, maxLines, maxBytes int) []byte {
 	}
 	return data
 }
+
+// ParseValueWithUnit parses a value like 500kB and returns the number
+// in bytes. The case of the unit will be ignored for user convenience.
+func ParseValueWithUnit(inp string) (int64, error) {
```

Comment:
The function name doesn't give a hint about what kind of value we have at hand (time, distance, length, etc). Perhaps _ParseByteSize_ or _ParseByteUnit_ or _ParseSizeUnit_?